src/components/Settings/SettingsMain.tsx

```diff
@@ -28,6 +28,7 @@ const messages = defineMessages({
   toastSettingsSuccess: 'Settings successfully saved!',
   toastSettingsFailure: 'Something went wrong while saving settings.',
   defaultPermissions: 'Default User Permissions',
+  useJellyfin: 'Use Jellyfin as Media Server',
```

Comment:
This setting shouldn't be necessary, since the user configured the media server when they set up Overseerr.

Reply:
Added this due to if the user wanted to _switch_ in the future.

Reply:
I think that introducing the option to switch will introduce lots of other issues.  I think we should table this for later.

Reply:
(I think we should remove this setting option until we are confident we can support such a switch.)

Reply:
Okay, sounds good. I'll remove the option.